roadmap/index.md

```diff
@@ -4,6 +4,38 @@ layout: default
 
 # Roadmap
 
+## 0.6.0 (August 2018)
+
+### Custom Resource Definitions support for Topic Operator
+
+The Topic Operator should have support for [Custom Resource Definitions (CRDs)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/).
```

Comment:
It's not so much "have support for CRDs" as "should use a Custom Resource instead of `ConfigMaps`"